wallet/src/main/java/com/android/identity_credential/wallet/ui/prompt/passphrase/PassphraseEntryField.kt

```diff
@@ -42,28 +42,33 @@ import kotlinx.coroutines.launch
  * A composable for entering a passphrase or PIN.
  *
  * @param constraints the constraints for the passphrase.
- * @param checkWeakPassphrase if true, checks and disallows for weak passphrase/PINs and also
- *   shows a hint if this is the case.
+ * @param imeAction optional, the icon to show for the keyboard action button. If parameter is not
+ *      provided or is [null] (default case), then it uses [ImeAction.Done].
+ *      Note: only the [ImeAction.Done] is recognized to have a task that is executed when the
+ *      user taps on the keyboard's action button.
```

Comment:
Why do we need this? Want to avoid as this will make moving `PassphraseEntryField` into a Compose Multiplatform library more tricky.

Reply:
This is an optional parameter for specifying the icon of the action button on the keyboard, if it's not specified (such as in existing composables using `PassphraseEntryField`) `it resorts to use the default "Done".

Regarding Kotlin Multiplatform, this parameter will probably end up being very important to implementing the iOS side of the keyboard and customization of the "action button" for when there's a fixed length passphrase and if the user sees the action button as a "checkmark" they will be misled into thinking they can complete the passphrase prompt by tapping on the keyboard button, but the button won't do anything so the user will think our app is broken and yield to loss of trust -- versus, if we show an arrow (or maybe even remove the action button altogether on iOS - not possible on android) then the user will continue to trust the prompt and the prompt will behave how the user expects it to behave.